config.py

```diff
@@ -213,6 +213,61 @@
     ```
 """
 
+Bipolar_T = Tuple[str, str, str]
```

Comment:
I don't like parameters which is CamelCase and snake_case.

I would just do:

`eeg_bipolar_channels: Iterable[Tuple[str, str, str]] = []`

Reply:
> I don't like parameters which is CamelCase and snake_case.

Heh, TypeScript habit. Will change.